src/sentry/api/urls.py

```diff
@@ -341,6 +343,7 @@
         GroupNotesDetailsEndpoint.as_view(),
     ),
     url(r"^(?P<issue_id>[^\/]+)/hashes/$", GroupHashesEndpoint.as_view()),
+    url(r"^(?P<issue_id>[^\/]+)/reprocessing/$", GroupReprocessingEndpoint.as_view()),
```

Comment:
It seems that we currently have such actions on index endpoints (e.g. `/issues/<id>/?reprocess=true`). That said, since we want to query for reprocessing status, it makes sense to introduce a new endpoint.

Reply:
yeah that was the idea PUT/POST for creating a "task entity" and GET on the same endpoint (or rather a subpath) for polling